backends/graphics3d/openglsdl/openglsdl-graphics3d.cpp

```diff
@@ -632,6 +664,17 @@ OpenGL::FrameBuffer *OpenGLSdlGraphics3dManager::createFramebuffer(uint width, u
 }
 
 void OpenGLSdlGraphics3dManager::updateScreen() {
+#if defined(USE_IMGUI) && SDL_VERSION_ATLEAST(2, 0, 0)
+	ImGui_ImplOpenGL3_NewFrame();
```

Comment:
This also would be run unconditionally for every game and every engine. I think, we should add some kind of a flag like "enableImgGui()" to the backend